backend/db/migrations/V0.22.2__CE-771.sql

```diff
@@ -0,0 +1,81 @@
+CREATE TABLE public.gir_type_code (
+    gir_type_code varchar(10) NOT NULL,
+    short_description varchar(50) NOT NULL,
+    long_description varchar(250) NULL,
+    display_order int4 NOT NULL,
+    active_ind bool NOT NULL,
+    create_user_id varchar(32) NULL,
+    create_user_guid uuid NULL,
```

Comment:
This is the second ticket I've seen someone trying to put a guid in our audit fields when it was not included in the ticket specification.  

Where did you get this field definition from?

Reply:
This was a straight cut and paste from V0_init.sql where I found the similiar species_code table and I used that as an example, I can see now the standard list of fields was updated after that initial table creation.

Reply:
Updated now to remove the GUID field.